x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/evaluate_condition.ts

```diff
@@ -16,13 +16,26 @@ import { InfraSource } from '../../sources';
 import { calculateFromBasedOnMetric } from './lib/calculate_from_based_on_metric';
 import { getData } from './lib/get_data';
 
+interface AdditionalContextField {
+  [x: string]: any;
+}
+
+export interface AdditionalContext {
+  cloud?: AdditionalContextField;
+  host?: AdditionalContextField;
+  container?: AdditionalContextField;
+  orchestrator?: AdditionalContextField;
+  labels?: AdditionalContextField;
+  tags?: AdditionalContextField;
+}
+
 export type ConditionResult = InventoryMetricConditions & {
```

Comment:
nit:
```
export type ConditionResult = InventoryMetricConditions & AdditionalContext & {
  shouldFire: boolean;
  shouldWarn: boolean;
  currentValue: number;
  isNoData: boolean;
  isError: boolean;
};
```

Reply:
I have added `context` field in the `ConditionResult` object now.